scipy-ar-2013/category.json

```diff
@@ -0,0 +1,4 @@
+{
+  "title": "SciPy Ar 2013",
```

Comment:
I wonder if this should not go next to the existing `scipy-la-2016`. What do you think @akielbowicz